app/views/static_pages/contact.html.erb

```diff
@@ -1,6 +1,6 @@
-<% provide(:title, "Contact") %>
+<% provide(:title, t(".title_page")) %>
```

Comment:
remove ()

Reply:
![image](https://user-images.githubusercontent.com/18642508/43375517-c8bc0fc2-93df-11e8-818f-2c149b24d6ed.png)
I can't remove it, it will return an error, how to resolve this

Reply:
oh i caught it, your mean just the () outside